lib/dictionaryloader.rb

```diff
@@ -0,0 +1,8 @@
+class DictionaryLoader
+  attr_accessor :dictionary_array
+  def initialize(filename)
+    File.open(filename, "r+") do |file|
+      @dictionary_array = file.readlines.map{|line| line.strip}
+    end
+  end
+end
```

Comment:
I can't get 5desk.txt to open for me on my own machine. You can call #readlines directly on the File, e.g. `File.readlines(filename)`